src/data/queries/personProfile.ts

```diff
@@ -9,41 +9,40 @@ import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
 import { NodePersonProfile } from '@/types/drupal/node'
 import { drupalClient } from '@/lib/drupal/drupalClient'
 import { PersonProfile } from '@/types/formatted/personProfile'
+import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes'
+import { entityBaseFields } from '@/lib/drupal/query'
 
-// Define the query params for fetching node--news_story.
-export const params: QueryParams<null> = () => {
-  return new DrupalJsonApiParams().addInclude([
-    'field_media.image',
-    'field_office',
-    'field_complete_biography',
-  ])
-}
+// // Define the query params for fetching node--person_profile.
```

Comment:
Is this query & data loader commented out only because we haven't officially gone through and (re-)built this page type?

Reply:
Yes, mostly. I originally commented it out because it (along with a few other resource types) weren't fully aligned with the others that we are building (e.g. missing properties). As a result, type errors were being raised in places referencing `FormattedResource` (e.g. `[[...slug]].tsx` default export - the template). So, initially, I started to try to fix these files, and `personProfile` was the first one. I realized things were not set up correctly with the query or the formatter, so I commented out the query (not needed currently) and fixed the formatter.

But, ultimately, I ended up introducing `FormattedPageResource` to differentiate between nodes that generate pages and nodes that are only entity references (for now, at least). This made it unnecessary to update all the other files accordingly, and allowed me to just "move on". There are still some resources that are in the same situation as `personProfile`, but they weren't addressed here. Specifically, the query is likely unnecessary (it can probably just be deleted rather than commented in `personProfile`) because, although it's a node, we aren't going to query it individually. In this way, it's more like a paragraph than a page-generating node.

Reply:
Thank you for the explanation, that does make sense. I do want to note that `personProfile` will generate pages at some point (though to your point, there are also profile paragraphs that do not generate pages). 
e.g. https://www.va.gov/butler-health-care/staff-profiles/sharon-coyle/